packages/core/src/__tests__/feature-app-manager.test.ts

```diff
@@ -15,7 +15,7 @@ interface MockFeatureServiceRegistry extends FeatureServiceRegistryLike {
 
 describe('FeatureAppManager', () => {
   let featureAppManager: FeatureAppManagerLike;
-  let mockRegistry: MockFeatureServiceRegistry;
+  let mockFeatureServiceRegistry: MockFeatureServiceRegistry;
```

Comment:
I didn’t change the name of this variable because the property of the Festure App manager is still called `registry`.

Reply:
Then we should rename the argument of the `FeatureAppManager` to `featureServiceRegistry`. Then it can stay that way. Is also only consistent.

Reply:
Yes, maybe that is a better idea.